### ISSUE TITLE
WARN-OpenLyrics: Received musixmatch search result but body was malformed: {"message":{"header":{"status_code":401,"execute_time":0.001,"hint":"renew"}}}

### ISSUE THREAD

**Issue by MegaBidoof** [jacquesh/foo_openlyrics#284]
When attempting to search for lyrics, with Musixmatch as one of the sources, I get the following error, and no lyrics are retrieved from Musixmatch:
`WARN-OpenLyrics: Received musixmatch search result but body was malformed: {"message":{"header":{"status_code":401,"execute_time":0.001,"hint":"renew"}}}`

I believe 401 is an authorization error, probably related to my Authentication Token. So it probably is just due to me having put the wrong token in the "Musixmatch Authentication Token" field.

The 2 different tokens I tried inputting on the "Musixmatch Authentication Token" field were:
1. API key, obtained directly from Musixmatch by creating a developer account.
2. "musixmatchUserToken", obtained by logging in with a Community account, enabling Web developer tools, going to the "Network" tab and selecting a request which has "musixmatchUserToken" or "usertoken" in the "Header" or "Cookie" tabs.

I would appreciate your help in getting a correct Authentication Token. I have searched online but found no other methods that should work.
I'm sorry for my lack of knowledge.

#### Steps to reproduce
1. Open console
2. Bulk search for lyrics

or

1. Open console
2. Manually search for lyrics

or

1. Open console
2. Play a song while having the OpenLyrics Panel in the UI.

#### Expected behavior

Lyrics are retrieved from Musixmatch, along with the other sources.

#### Versions

* foobar2000 version: 1.6.7
* OpenLyrics version: 1.6

**Comment by jacquesh** [comment]
Just as a baseline, can you try with an authentication token retrieved by openlyrics (click on the "?" button next to the token field in the openlyrics preferences)?

**Comment by MegaBidoof** [comment]
First of all, apologies for the late reply. Yesterday you commented right after I had left home, and I returned late.

Now, onto what really matters. Using the "?" button I was able to get a token which works, and now I am able to retrieve the lyrics from Musixmatch, so thank you.

The reason I hadn't tried that before was because that button was outside of the window, and so not visible. This happens because, since I'm on a laptop, the default display scale is 125%, which leads to the button not fitting inside the "Preferences" window, which is not resizable.
![Preferences - 125%](https://github.com/jacquesh/foo_openlyrics/assets/119820017/33b13abb-92a0-484f-93bc-c9d46880ef9b)
![Scale 125%](https://github.com/jacquesh/foo_openlyrics/assets/119820017/90fa6eb1-8c19-4c91-91a8-0276e8d75522)


Your comment made me realize there was probably a button I could not see because it was outside the window. After changing the scale to 100% and restarting Foobar, I was finally able to see the button, and get a usable token.

![Preferences - 100%](https://github.com/jacquesh/foo_openlyrics/assets/119820017/6bd93be4-3720-4f25-b7b0-083ce1bdc333)
